### PR TITLE
1.0

### PR DESCRIPTION
**Pull request by jonwinton**
v1 PR 🎉  🌮

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  dirname = __dirname.split('/').pop(),
+  lib = require('./'),
+  expect = require('chai').expect;
+
+describe(dirname, function () {
+  let sandbox,
+    pipeSpy = sinon.spy(),
+    childSpy = sinon.spy(),
+    fakeLog = createFakeLogger();
+
```
Comment (amycheng) [inline test.js]: extra line break

File: index.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+var pino = require('pino'), // Can be overwritten for testing
```
Comment (amycheng) [inline index.js]: I'm curious as to why you chose `pino` over `winston`, which I've seen used in other clay/amphora-related projects?

Comment (jonwinton) [reply index.js]: http://getpino.io/#/?id=benchmarks

Comment (amycheng) [reply index.js]: EXTREME

Comment (jonwinton) [reply index.js]: It's a multipart answer. 

1. Pino is more performant in general
2. Winston accommodated transports as a main part of the primary process thread whereas Pino relies on piping to stdout and then using another process to transport. It's more performant and flexible.
3. Pino is configured with tools like ELK in mind and then pretty printing as a secondary priority

File: index.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+ *
+ * @param  {Object} args
+ * @return {Function}
+ */
+function init(args) {
+  if (!args || !Object.keys(args).length || !args.name) {
+    throw new Error('Init must be called with `name` property');
+  }
+
+  let { name = '', prettyPrint = false, meta = undefined } = args,
```
[the remark this answers is not in the document]
Comment (jonwinton) [reply index.js]: Fixed

File: README.md [hunk 4]
```diff
@@ -1 +1,44 @@
...
+  - Kiln
+  - Kiln plugins (Clay Space Edit)
+  - `model.js` files
+
+## API
+
+Instantiate the `clay-log` module.
+
+```
+EXAMPLE
```
Comment (nelsonpecora) [inline README.md]: ...example tho?

File: README.md [hunk 5]
```diff
@@ -1 +1,44 @@
...
+```
+
+Pretty printing is controlled by an environment variable
+```
+EXAMPLE PRETTY VS NON-PRETTY PRINTING
```
Comment (nelsonpecora) [inline README.md]: beep boop

File: README.md [hunk 6]
```diff
@@ -1 +1,44 @@
...
+```
+
+Attach metadata associated with each line. Handy for environment, version number, etc.
+
+```
+ADD META API
```
Comment (nelsonpecora) [inline README.md]: how do I computer tho

**Pull request merged**